clang/test/CodeGenSYCL/uses_aspects.cpp

```diff
@@ -28,7 +28,7 @@ template <sycl::aspect Aspect>
 void func5() {}
 
 [[__sycl_detail__::__uses_aspects__(sycl::aspect::cpu)]] void func6();
-// CHECK: define dso_local spir_func void @{{.*}}func6{{.*}} !sycl_used_aspects ![[ASPECTS1]] {
+// CHECK: define dso_local spir_func void @{{.*}}func6{{.*}} !sycl_used_aspects ![[ASPECTS4:[0-9]+]] {
```

Comment:
The reason this is needed now is because the aspect propagation pass is enabled in all cases, including for this test. It will apply the aspect uses from the body onto the function, so instead of it using only `sycl::aspect::cpu` it will also use `sycl::aspect::fp16` as it is propagated from `Type2WithAspect`.